### PR TITLE
php 8.1 fix

### PR DESCRIPTION
**Pull request by chris18890**
php fatal error array and string offset access syntax

https://stackoverflow.com/questions/59158548/array-and-string-offset-access-syntax-with-curly-braces-is-deprecated for an example of the issue

### CODE REVIEW

**Comment by chris18890** [comment]
Can I get a review on this @ramgarden / @elplatt

File: crm/modules/user_meta/user_meta.inc.php [hunk 1]
```diff
@@ -184,7 +184,8 @@ function user_meta_data ($opts = array()) {
         ";
     }
     if (!empty($opts['cid'])) {
-        $esc_cid = mysqli_real_escape_string($db_connect, $opts['cid']);
+        $cid = $opts['cid'][0];
+        $esc_cid = mysqli_real_escape_string($db_connect, $cid);
```
Comment (ramgarden) [inline crm/modules/user_meta/user_meta.inc.php:188]: nice job reducing the cognitive complexity here!

**Review by ramgarden — APPROVED** [review]
Fixed the curly brace to square bracket issue and reduced the cognitive complexity a bit.  approved!

File: crm/modules/user_meta/user_meta.inc.php (hunk 1 above)
Comment (chris18890) [reply crm/modules/user_meta/user_meta.inc.php:188]: TBH I preferred the 1 liner, PHP 7.4 didn't mind the 1 liner, PHP 8.1 hated the 1 liner :( :P

**Pull request merged**